Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -219,14 +219,22 @@ void VertexShaderManager::SetConstants()
       dstlight.pos[1] = light.dpos[1];
       dstlight.pos[2] = light.dpos[2];
 
+      // TODO: Hardware testing is needed to confirm that this normalization is correct
```

Comment:
Do you plan on doing hardware testing before this is merged?

Reply:
No.  I plan on doing the hardware testing eventually, but it would be part of a separate, larger PR that targets more than just fixing Mesa on FifoCI.